src/Matches.js

```diff
@@ -1,136 +1,151 @@
-            if (this.state.userIds[i] != prevState.userIds[i]) {
-                this.get_matches()
-                break;
-            }
-        }
-        
-    }
-
-    render(){
-        let matchingSection;
-        if (this.state.matchesExist == "exists"){
-            let userItems = [];
-            for (var i = 0; i < this.state.userIds.length; i++){
-                var pos_user = this.state.userIds[i];
-                userItems.push(
-                    <button className='pos-user' 
-                            key={pos_user} 
-                            value = {this.state.userIds[i]+"|"+this.state.firstnames[i]} 
-                            onClick = {this.selectUser}>
-                        {this.state.firstnames[i]}
-                    </button>  
-                )
-                
+  }
 
-            }
-            matchingSection = <h3 id='Matches-congrats'>
-                <p>Congratulations,</p>
-                <p>you have a match!</p> 
-                <p>
-                    {userItems}
-                </p>
-                </h3>
-        }
-        else if (this.state.matchesExist == "not exists") {
-            matchingSection = <h2 id='Matches-none'>
-                <p>Sorry, no one met the matching criteria you set.</p> 
-                <p>We suggest you to edit your profile, or wait for more users to join our community.</p> 
-                <p>Please try again later :(</p>
-            </h2>
-        }
-        else {
-            matchingSection = <h2></h2>
-        }
-        return(
-            <div id='Matches-section'>
-                 {matchingSection}                        
-
-                <div class = "swipingButton" id = 'swipingButton'>
-                    <Link to={{pathname: '/main', state: {id: this.state.userId}}}>Keep Swiping</Link>
-                </div>
-                <div class = "viewProfileButton" id = 'viewProfileButton'>
-                    <Link to={{pathname: '/viewprofile', state: {id: this.state.userId}}}>View Profile</Link>
-                </div>
-            </div>
-        )
+  render() {
+    let matchingSection;
+    if (this.state.matchesExist == "exists") {
+      let userItems = [];
+      for (var i = 0; i < this.state.userIds.length; i++) {
+        var pos_user = this.state.userIds[i];
+        userItems.push(
+          <button
+            className="pos-user"
+            key={pos_user}
+            value={this.state.userIds[i] + "|" + this.state.firstnames[i]}
+            onClick={this.selectUser}
+          >
+            {this.state.firstnames[i]}
+          </button>
+        );
+      }
+      matchingSection = (
+        <h3 id="Matches-congrats">
+          <img src="happy-penguin.svg"></img>
+          <p>Congratulations, you have a match! </p>
+          <div className="matchingBox">
+            <br></br>
+            <h2>{userItems}</h2>
+            <br></br>
+          </div>
+        </h3>
```

Comment:
Please note this section also relies on Melody's section. This should be done after her backend is completed so the styling can be done based on that. Please discuss with Melody.